### ISSUE TITLE
Salut c'est zuzulo

### ISSUE THREAD

**Issue by finkiki** [F-Horowitz/aoehexagone_files#1]
On va rester en contact sur github, c'est plus sain. Mes comptes JVC foutent le camp le uns après les autres.

**Comment by F-Horowitz** [comment]
C'est ok. On devrait inviter les autres aussi. Tu as des nouvelles de Carnage par hasard ?
Sinon je peux savoir pourquoi tes comptes JVC "foutent le camp les uns après les autres" ? Tout de même, pour être obligé de se réfugier sur Github à cause de ta situation, il y a quelque chose qui va pas.

On était bien sur JVC. Enfin, y a heroscaper qui vient certainement sur Git pour utiliser le code source donc forcément.
**_HEROSCAPER, N'HÉSITE PAS A VENIR ICI SI JAMAIS TU LIS CES LIGNES! IDEM POUR TOI CARNAGE!_**

**Comment by finkiki** [comment]
Nan, pas de nouvelles de Carnage. Sinon, je vais jeter un oeil sur les fichiers. Donc définitivement, tu ne veux pas utiliser de CMS ? 
On part sur un truc orienté rétro rétro rétro ?

**Comment by F-Horowitz** [comment]
Oui, on part définitivement sur du rétro rétro rétro. C'est prévu depuis le départ de toute façon, à mes yeux le CMS c'est pourri. Pas assez personnalisables. De toute manière tu es rédacteur priviligié à la base donc ce n'est pas censé te déranger....

**Comment by finkiki** [comment]
Oui enfin je me débrouille bien en Html et css hein, tu te rappelle l'essai de site que je t'avais envoyé ? ça pétait le feu. 

Tu l'as encore ce site ?  J'ai perdu les fichiers je crois bien.

**Comment by Foksdu** [comment]
Retro = web statique du coup, si on envoie une info va falloir recharger la page si j'ai bien compris, et pas actualiser au fur et à mesure.

J'aimerais bien savoir quand même ce que comprendra le site, des sections particulières ? etc.
Et sinon la BDD vide je l'aurais quand ?

**Comment by F-Horowitz** [comment]
Zuzulo malheureusement je ne suis pas sûr d'avoir encore les fichiers. Par contre il se peut qu'il soit toujours sur la conv JVC.

Hero la BDD je te l'ai envoyée sur JVC normalement. En mediafire.  je vais essayer de rédiger une petite présentation technique

**Comment by finkiki** [comment]
Il te faut une BDD pourquoi ? Tu veux permettre l'inscription de membres c'est ça ?

**Comment by Foksdu** [comment]
Par exemple, et pour les tests ça évitera aussi d'avoir une énorme erreur sql en plein milieu de la page.
Ah et oui, j'avais oublié que tu ne la mettais pas ici funkumo, désolé de l'étourderie.

**Comment by F-Horowitz** [comment]
Pas grave.
Pour la présentation du projet je l'ai déjà faite en individuel mais elle n'est pas adaptée à la publication même à espace privée... Je vais me charger de cela.

**Comment by Foksdu** [comment]
J'my mets (s'rait temps). Je vais essayer d'être régulier et d'avancer par étapes donc je n'ai aucune idée du temps que ça va prendre même si ça m'a l'air d'être un gros projet au vu de la bdd (d'ailleurs tu t'es bien fait plaisir dedans haha).

**Comment by Foksdu** [comment]
J'aurais besoin des templates stp !

include('../motor/config.php');
include('../motor/constantes.php');
include('../motor/fonctions.php');
include_once('cadre_membre.php');

Sinon j'ai une chiée d'erreur quoi.

**Comment by F-Horowitz** [comment]
Je te les passe vendredi, normalement. PRENDS-EN SOIN. >:(
Et oui je me suis fait plaisir :noel: (et encore elle n'est pas complète, c'est un proto)

**Comment by Foksdu** [comment]
Si j'avais tout eu plus tôt j'aurais déjà pu bien entamer le projet là :'(
Toujours pas moyen de tout avoir d'un coup malgré ta paranoïa ?

Vu que je suppose qu'il y a d'autres fichiers en réserve qui vont me servir et que j'aurais pas. On gagnerait un temps précieux !

**Comment by F-Horowitz** [comment]
Bon j'essaierais de tout te passer vendredi

**Comment by Foksdu** [comment]
On est vendredi soir :p

**Comment by F-Horowitz** [comment]
Coucou. Tout est dit dans le dernier upload.

**Comment by F-Horowitz** [comment]
Euh.. allo la Terre ???

**Comment by F-Horowitz** [comment]
HÉ NE LAISSEZ PAS LE PROJET COULER!
Vous recevez pas les mails ou quoi ?!

**Comment by finkiki** [comment]
Ce fil est là pour ça. Dis-moi ce qu'il y a ?

**Comment by F-Horowitz** [comment]
Il y a 2sem j'ai uploadé un ZIP contenant le GDD du projet Hexagone et l'essentiel des fichiers du site. Vous y trouverez ce qu'il faut pour vous, je l'espère.

Petite anecdote : j'ai changé de PC (l'autre est défectueux) pour les fichiers du site. Les prochains screens seront sous W7. Voilà. :D

**Comment by Foksdu** [comment]
Oups, les notif' étaient désactivées jusqu'à très récemment sur mon téléphone (je vais jamais voir mes emails sur ordi), je me mets sur le projet dès demain avec le ZIP. Je vais essayer de te tenir au courant des avancées au moins 1 fois par semaine.

**Comment by F-Horowitz** [comment]
Entendu.

**Comment by F-Horowitz** [comment]
Hé mine de rien, ça vous dirait pas qu'on utilise un moyen de discussion plus instantané un jour, comme Skype (sans webcam, anonymat avant tout) ou Discord ? Comme ça il n'y aurait peut-être moins besoin d'attendre plusieurs jours avant que son interlocuteur ne reçoive le message et réponde.

**Comment by F-Horowitz** [comment]
Du nouveau ?

**Comment by F-Horowitz** [comment]
ok, ça veut tout dire...

**Comment by Wolf-silver** [comment]
11

**Comment by finkiki** [comment]
24

**Comment by F-Horowitz** [comment]
30

**Comment by F-Horowitz** [comment]
7
C'est aussi vide que le menu d'AOE2 ici..

**Comment by finkiki** [comment]
Désolé mais je n'ai pas de temps libre en ce moment et cet été impossible aussi.

**Comment by F-Horowitz** [comment]
Compréhensible, bon moi je continue pendant ce temps. Au fait Heroscaper s'est barré.

**Comment by F-Horowitz** [comment]
Mais j'y pense, tu seras dispo quand en fait ?
